packages/text-annotator/src/model/w3c/W3CTextFormatAdapter.ts

```diff
@@ -1,45 +1,60 @@
-import { v4 as uuidv4 } from 'uuid';
+import { v4 as uuidv4 } from "uuid";
```

Comment:
The single quotes are used across the repo and changing them will yield quite a few conflicts